infra/aws/terraform/registry.k8s.io/s3/bucket.tf

```diff
@@ -60,29 +52,15 @@ resource "aws_s3_bucket_policy" "registry-k8s-io-public-read" {
   })
 }
 
-resource "aws_iam_user_policy" "registry-k8s-io-rw" {
-  name = "${aws_s3_bucket.registry-k8s-io.bucket}-access"
-  user = var.iam_user_name
```

Comment:
note to reviewers: this IAM User policy should have been removed in an earlier commit (see PR #3648) but we only noticed it just now.